packages/wrangler/src/pages.tsx

```diff
@@ -1067,3 +1113,31 @@ const invalidAssetsFetch: typeof fetch = () => {
     "Trying to fetch assets directly when there is no `directory` option specified, and not in `local` mode."
   );
 };
+
+const listProjects = async ({
+  accountId,
+}: {
+  accountId: string;
+}): Promise<Array<Project>> => {
+  const pageSize = 10;
+  let page = 1;
+  const results = [];
+  while (results.length % pageSize === 0) {
+    const json: Array<Project> = await fetchResult(
+      `/accounts/${accountId}/pages/projects`,
+      {},
+      new URLSearchParams({
+        per_page: pageSize.toString(),
+        order: "title",
```

Comment:
The only params this endpoint accepts are `page`, `per_page` and `name`.

Reply:
Order is fixed as last modified desc.

Reply:
Fun. Thanks for pointing that out!